.github/compatibility-test-matrices/main/genesis-chain-a.json

```diff
@@ -0,0 +1,17 @@
+{
+  "chain-a": [
+    "main"
+  ],
+  "chain-b": [
+    "main"
+  ],
+  "entrypoint": [
+    "TestGenesisTestSuite"
+  ],
+  "test": [
+    "TestIBCGenesis"
+  ],
+  "relayer-type": [
+    "rly"
```

Comment:
it looks like all of the new files are using `rly` but we've updated to use `hermes` as the default everywhere else. I think we should make sure we're consistent. (down the line we can maybe run both here?)

Reply:
oh, yes. Should I change to `hermes` only for `release/v8.0.x` or also for all the previous release lines?

Reply:
I think we can do it for everything now, and then re-add rly later on.

Reply:
I switched to hermes everywhere.